regnety/make_tfrecords.py

```diff
@@ -4,15 +4,24 @@
 
 def main():
     parser = argparse.ArgumentParser(description="Make TFRecords")
-    parser.add_argument("-o", "--odir", type=str,)
-    parser.add_argument("-d", "--data_dir", type=str)
-    parser.add_argument("-fpr", "--file_prefix", type=str)
-    parser.add_argument("-synpath","--synset_filepath", type=str, default='')
-    parser.add_argument("-bs", "--batch_size", type=int, default=1024)
-    parser.add_argument("-f", "--log_freq",type=int, default=50)
-    parser.add_argument("-s", "--shuffle", action='store_true')
-    parser.add_argument("-v", "--validation_set",action='store_true')
-    parser.add_argument("-b", "--use_apache_beam",action='store_true')
+    parser.add_argument("-o", "--odir", type=str,
+        help="Output directory for TFRecords.")
+    parser.add_argument("-d", "--data_dir", type=str,
+        help="Input data directory. Must have ImageNet-like directory structure.")
```

Comment:
If we could provide a link to how ImageNet-like folder structure looks like it'd be great. But don't sweat it.